pkg/pillar/types/contenttreetypes.go

```diff
@@ -26,6 +26,7 @@ type ContentTreeConfig struct {
 	GenerationCounter int64
 	DisplayName       string
 	CustomMeta        string
+	IsAppImage        bool
```

Comment:
I'm guessing this is to tell apart infrastucture pieces like the k3s, longhorn, etc pieces.
Or is it to tell apart app images and baseos images??
If the former it would make sense to pick a different name and explain in a comment.

Reply:
This is for the eve app image. Nothing to do with the kubernetes components.

Reply:
Why does the code in volumemgr now need to care about app images vs. baseos images? Where are the comments explaining this?

Reply:
I will add comments. Basically since we will be supporting native container launch we need prepend the docker.io prefix to the images we are planning to launch as native containers.  The prefix is applied in volumegr/handlework.go:mayAppendImgPrefix(). But to apply the prefix we need to know if the image is an eve container app or not. We now keep that information in ContentTreeConfig.

Reply:
Don't we have IsContainer() for this?

Reply:
Yes good point. I am looking into that with Naiming and will update.

Reply:
Fixed this code and removed the flag IsAppImage and used IsContainer() wherever needed. Submitted a new commit.